### PR TITLE
openEuler 22.03-SP1 released, so we switch to latest repo url

### PR DESCRIPTION
**Pull request by pkking**
As newer 22.03 LTS SP1 released, switch to latest release.

Signed-off-by: Li Chaoran <pkwarcraft@gmail.com>

### CODE REVIEW

File: mock-core-configs/etc/mock/templates/openeuler-22.03.tpl [hunk 1]
```diff
@@ -27,44 +27,47 @@ protected_packages=
 module_platform_id=platform:oe2203
 user_agent={{ user_agent }}
 
+# all openEuler LTS release will continue developing and releasing SPx version
+# such as 22.03-LTS -> 22.03-LTS-SP1 -> 22.03-LTS-SP2 ...
+# all LTS-SPx is compatible with its LTS release
```
Comment (praiskup) [inline mock-core-configs/etc/mock/templates/openeuler-22.03.tpl:32]: Humble feedback :-) can't you have a server-side symbolic links?  Say `openEuler-22.03-latest` pointing at `openEuler-22.03-LTS-SP1`, later when available pointing at `22.03-LTS-SP2`, etc..?  That way your users can keep using the same `*-latest` baseurl= option (including Mock).

**Comment by praiskup** [comment]
Otherwise +1.

File: mock-core-configs/etc/mock/templates/openeuler-22.03.tpl (hunk 1 above)
Comment (pkking) [reply mock-core-configs/etc/mock/templates/openeuler-22.03.tpl:32]: Big agree with you, its also a maintaince burden for us, will make a suggestion to our release team :)

Comment (pkking) [reply mock-core-configs/etc/mock/templates/openeuler-22.03.tpl:32]: > Humble feedback :-) can't you have a server-side symbolic links? Say `openEuler-22.03-latest` pointing at `openEuler-22.03-LTS-SP1`, later when available pointing at `22.03-LTS-SP2`, etc..? That way your users can keep using the same `*-latest` baseurl= option (including Mock).

Big agree with you, it's also a burden for us, will make a discussion with release team 👍

**Comment by pkking** [comment]
@Yikun @Tommlike please help to review

File: mock-core-configs/etc/mock/templates/openeuler-22.03.tpl (hunk 1 above)
Comment (TommyLike) [reply mock-core-configs/etc/mock/templates/openeuler-22.03.tpl:32]: +1

Comment (praiskup) [reply mock-core-configs/etc/mock/templates/openeuler-22.03.tpl:32]: Anyway, thank you for explaining the release cycle here!

Does the +1 mean this is reviewed?

Comment (pkking) [reply mock-core-configs/etc/mock/templates/openeuler-22.03.tpl:32]: Yes, please make ti merged.

**Pull request merged**